src/components/side-by-side/side-by-side.scss

```diff
@@ -14,5 +16,43 @@
     z-index: 1;
     padding: 16px;
     flex-direction: row-reverse;
+    animation: appears $animation-duration linear;
+  }
+}
+
+.animatedScale {
+  animation: scaling $animation-duration linear;
+}
+
+.animatedFade {
+  animation: fading $animation-duration linear;
+}
+
+@keyframes appears {
+  from {
+    visibility: hidden;
+  }
+  to {
+    visibility: hidden;
+  }
```

Comment:
from hidden to hidden? Isn't it a bug?

Reply:
it makes a trick with buttons that should be hidden during animation